backend/copr_backend/pulp.py

```diff
@@ -485,7 +499,7 @@ def delete_repository(self, repository):
         """
         url = self.config["base_url"] + repository
         self.log.info("Pulp: delete_repository: %s", repository)
-        return requests.delete(url, **self.request_params)
+        return self.send("DELETE", url)
```

Comment:
I just realized that `SafeRequest` doesn't know how to handle "DELETE" ... I need to fix this. But otherwise the PR is ready for review